### DISCUSSION TITLE
Add a navigation option to the Separator/Divider card

### DISCUSSION THREAD

**Discussion by FrankJaco** [Clooos/Bubble-Card#369] [Feature requests]
I think this could be a quick win as it should be easy to add _(well, at least I think it should be easy)_.

I would like to see the ability to click on the icon and/or title of a Separator and have it navigate to another view.   Like many people, I have sub-views for things like "all lights in the house", and other sub-views that show "all devices in a room" etc.  Below is a snippet from my whole house Lights view:

![image](https://github.com/Clooos/Bubble-Card/assets/79318542/1884dbfe-b022-4630-9c76-3df5e2d7061c)

I would like the ability to be able to click on the Sofa icon and/or title (Living Room) and have it automatically navigate to my Living Room sub-view.

Thx,

fj

**Comment by bodydoc** [comment]
Hello,

As @FrankJaco suggested, my entire Dashboard is build around sub-views. Separator from Bubble card is amazing, stylish and simple. Only thing missing is the option to navigate from it.
+1 on this request. 

Thank you in advance!

**Comment by Clooos** [comment]
It will finally be a new feature in the next release 🍻

**Comment by Nico87LBS** [comment]
Hello, 
Is this feature implemented ? I can't see anything in the doc or in the card editor.
It would be great to have it to replace the heading card.
Thanks

**Reply by Clooos** [reply]
Hi! It is now possible with sub-buttons since v2, but indeed a lot of you want to be able to tap on it directly, and I will add this possibility at some point 👌

**Reply by Nico87LBS** [reply]
Thanks,
Yes sub-buttons are great but as many, I would like to be able to click on the title itself, like the heading card in home assistant.
Thanks for your work, bubble card is great 💪👍
I will wait for this option.
